ui/tests/integration/components/open-api-explorer/swagger-ui-test.js

```diff
@@ -18,6 +18,22 @@ const SELECTORS = {
   searchInput: 'input.operation-filter-input',
   apiPathBlock: '.opblock-post',
   operationId: '.opblock-summary-operation-id',
+  controlArrowButton: '.opblock-control-arrow',
+  copyButton: '.copy-to-clipboard',
+  tryItOutButton: '.try-out button',
+};
+
+// for some reason search filtering does not update with ember test helpers
+// possibly due to swagger-ui event implementation
+// using native window fn to workaround
+const setNativeInputValue = (value) => {
```

Comment:
seems funky but if it works, fine with me! especially since this is just test code